main.py

```diff
@@ -11,28 +13,35 @@
 BULLETS_VELOCITY = 10
 RED = (255, 0, 0)
 
-SPACESHIP_IMAGE = pygame.image.load(os.path.join("Assets", "spaceship_red.png"))
-SPACESHIP = pygame.transform.rotate(
-    pygame.transform.scale(SPACESHIP_IMAGE, (SPACESHIP_WIDTH, SPACESHIP_HEIGHT)), 180
+SPACESHIP = Player(
+    "Player1",
+    "spaceship_red.png",
+    SPACESHIP_WIDTH,
+    SPACESHIP_HEIGHT,
+    angle=180,
+    hp=100,
 )
 
-ENEMY_IMAGE = pygame.image.load(os.path.join("Assets", "spaceship_yellow.png"))
-ENEMY = pygame.transform.rotate(
-    pygame.transform.scale(ENEMY_IMAGE, (SPACESHIP_WIDTH, SPACESHIP_HEIGHT)), 0
+
+ENEMY = Player(
+    "Enemy1",
+    "spaceship_yellow.png",
+    SPACESHIP_WIDTH - 3,
+    SPACESHIP_HEIGHT - 3,
+    angle=0,
+    hp=50,
 )
 
 SPACE_IMAGE = pygame.image.load(os.path.join("Assets", "space.png"))
-SPACE = pygame.transform.rotate(
-    pygame.transform.scale(SPACE_IMAGE, (WIDTH, HEIGHT)), 180
-)
+SPACE = pygame.transform.rotate(pygame.transform.scale(SPACE_IMAGE, (WIDTH, HEIGHT)), 0)
```

Comment:
Oh we can change this to use the load_png function. 

I'll knock that out really quick.